test/index.js

```diff
@@ -11,18 +11,29 @@ describe('unique stream', function() {
     s.readable = true;
 
     var n = 10;
-    var next = after(n, function () {
+    var next = after(n * 2, function () {
       setImmediate(function () {
         s.emit('end');
       });
     });
 
-    for (var i = 0; i < n; i++) {
-      var o = {
-        type: type,
-        name: 'name ' + i,
-        number: i * 10
-      };
+    for (var k = 0; k < n * 2; k++) {
+      var i = Math.floor(k / 2);
+      var o;
+      if (k % 2 === 0) {
+        o = {
+          type: type,
+          name: 'name ' + i,
+          number: i * 10,
+        };
+      }
```

Comment:
Updated. Is the style of this project documented somewhere? I didn't see anything in `CONTRIBUTING.md` or any linter config files.